### PR TITLE
Fix page redirect preview

### LINKED ISSUE #2380: Where to set up prefix redirect
## Details

* Project URL: docs.signalfx.com

I want to rename a file, and thus want to redirect from the old name to the new name. I have read the docs at http://docs.readthedocs.io/en/latest/user-defined-redirects.html, but it isn't telling me where I go to create that configuration. What file should I be editing to add the Type/From URL/To URL info?

Thanks -- Barbara

### PR DESCRIPTION
**Pull request by rixx**
It included '//' - this ought to fix #2380.

### CODE REVIEW

File: readthedocs/templates/projects/project_redirects.html [hunk 1]
```diff
@@ -22,8 +22,12 @@
                 redirect_target = "/$lang/$version/faq.html";
             }
             else if (option === "page" ) {
-                redirect_from = "/$lang/$version/" + field_from_url.val();
-                redirect_target = "/$lang/$version/" + field_to_url.val();
+                field_from = field_from_url.val();
+                field_to = field_to_url.val();
+                field_from_absolute = (field_from.length > 0) && (field_from[0] === "/");
+                field_to_absolute = (field_to.length > 0) && (field_to[0] === "/");
```
Comment (agjohnson) [inline readthedocs/templates/projects/project_redirects.html]: L27 + L28 are replicating `field_to.startsWith()` etc

File: readthedocs/templates/projects/project_redirects.html [hunk 2]
```diff
@@ -22,8 +22,12 @@
...
+                redirect_from = "/$lang/$version" + (field_from_absolute ? "" : "/") + field_from;
+                redirect_target = "/$lang/$version" + (field_to_absolute ? "" : "/") + field_to;
```
Comment (agjohnson) [inline readthedocs/templates/projects/project_redirects.html]: Conditional logic wouldn't be needed here if just using a simple regex to remove leading `/`. For example:

```javascript
redirect_from = "/$lang/$version/" + field_from_url.val().replace(/^\/+/, '')
```

Comment (rixx) [inline readthedocs/templates/projects/project_redirects.html]: Oh. yes, that's much cleaner, I'll edit the PR accordingly. Thank you for the review.

**Comment by vidartf** [comment]
It would be nice if this could be cleaned up an merged. Threw me off causing me almost half a day's work.

**Comment by stsewd** [comment]
@rixx hi, are you still interested on going with this PR? Would be awesome to got your contribution merged.

**Comment by RichardLitt** [comment]
@fenilgandhi That label isn't exclusive! We just tag some issues as easier. If you want to go through the backlog, I'm sure there are more issues you could fix. For now, I think this issue is already covered.

**Comment by vidartf** [comment]
Note: As the original author hasn't responded in almost a year, this PR could benefit from:
- Cloning this branch
- Adding one or more new commits fixing the review points
- Resubmitting as a new PR

**Comment by RichardLitt** [comment]
@vidartf If you'd like to do that, go ahead. :)

**Comment by stsewd** [comment]
@vidartf are you still interesting on doing the PR? Or maybe @rixx

**Comment by vidartf** [comment]
#3811.

**Pull request closed without merging**